### ISSUE TITLE
Web scrobbler error

### ISSUE THREAD

**Issue by beweress** [david-sabata/web-scrobbler#699]
I was trying to play music from VK and YouTube, but scrobbler type me this message:
![7b957a99b4 1](https://cloud.githubusercontent.com/assets/13871004/9359042/cf9412d8-4697-11e5-8730-256d27cbb406.png)
I was checking and everything is OK, I even reinstalled scrobbler but it's still not working.
I'm using version 1.50 now.

**Comment by inverse** [comment]
This: http://status.last.fm/

they are on the new site and are having huge problems.

**Comment by inverse** [comment]
dupe of #698

**Comment by Artyeshi** [comment]
Yes, VC does not work

**Comment by artzub** [comment]
request: https://ws.audioscrobbler.com/2.0/?method=track.updatenowplaying& ```many data```
answer: 
```
<lfm status="failed">
<error code="13">
Invalid method signature supplied
</error>
</lfm>
```

**Comment by bes-internal** [comment]
I parse: our web-scrobbler not working, and @david-sabata somewhere resting and does not want to help a ~cute~ girl

**Comment by inverse** [comment]
Could be resolved by #701

**Comment by david-sabata** [comment]
Yep, duplicate of #698. Affects all v1 connectors
